defaults/main.yml

```diff
@@ -79,6 +79,22 @@ apt_unattended_upgrades_automatic_reboot: no
 # Values: now | 02:00 | ...
 apt_unattended_upgrades_automatic_reboot_time: now
 
+# Override download timer ?
+apt_unattended_upgrades_download_timer_override: no
```

Comment:
There are a lot of variables that need or could be added so I'm thinking of rather replacing that with a dictionary conf object in case you want to override it... something like:

```yaml
# default value
apt_unattended_upgrades_download_timer_conf: null

# in case of override
apt_unattended_upgrades_download_timer_conf:
  OnCalendar: Mon..Fri *-*-* 6:00
  RandomizedDelaySec: ...
  Persistent: ...
```

This would mean just to dump the object into the template and making it more flexible to configurations.... 

A check in the tasks would then look like:

```
when: apt_unattended_upgrades_download_timer_conf is not none
```

Reply:
OK